server/services/database/profile.js

```diff
@@ -22,7 +22,28 @@ const getAllProfiles = () => {
 	return pool.query("SELECT * FROM profile").then((result) => result.rows);
 };
 
+const getProfileById = (profileId) => {
+	return pool
+		.query("SELECT * FROM profile WHERE profile_public_id=$1", [profileId])
+		.then((result) => result.rows);
+};
+
+const getEducationById = (profileId) => {
+	return pool
+		.query("SELECT * FROM education WHERE profile_id=$1", [profileId])
+		.then((result) => result.rows);
+};
+
+const getWorkExperiencesById = (profileId) => {
```

Comment:
I think the selects on work_experience should be in services/database/work_experiences.js